src/main/webapp/features/shared/services/game/Game.service.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+angular.module('catan')
+        .factory('Game', ['GameModel', '$q', 'Remote', function (GameModel, $q, Remote) {
+
+            var Game = function(idOrDetails) {
```

Comment:
wtf, what idOrDetails? it is very weird logic. as I understand, it is kind of constructor override, but it is better to make constructor take 3 parameters: id, details, status, so that we don't need strange names in code, like 'idOrDetails' and don't need this checks: typeof idOrDetails === "number" and typeof idOrDetails === "object"